vaadin-time-picker-flow-demo/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
```

Comment:
There's a dependency for `vaadin-combo-box-flow` defined on this POM and at the line `54` it sets the version to `3.1-SNAPSHOT`. This should be changed to the one we are bumping in combo-box. So, this PR is blocked by https://github.com/vaadin/vaadin-combo-box-flow/pull/372, as we need that snapshot first in order to proceed with this one.